artman/pipelines/gapic_generation.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
The answer to all of these is essentially: There is no way to tell git that a file was moved; it detects this (`git mv` does not actually save move metadata). I probably modified these four files sufficiently for git to not realize that it was a move as opposed to a deletion and addition.

It is probably _possible_ to go back in history and make this happen, but difficult.

More information: http://stackoverflow.com/questions/1430749/getting-git-to-acknowledge-previously-moved-files